packages/admin/src/FilamentManager.php

```diff
@@ -168,6 +175,21 @@ public function getGlobalSearchProvider(): GlobalSearchProvider
         return app($this->globalSearchProvider);
     }
 
+    public function renderHook(string $name): ?string
+    {
+        if (! array_key_exists($name, $this->renderHooks)) {
+            return null;
+        }
+
+        $output = '';
+
+        foreach ($this->renderHooks[$name] as $renderHook) {
+            $output .= (string) app()->call($renderHook);
```

Comment:
no improvement here, just TIL that view objects can be cast to strings to be rendered? 🤯

Reply:
They sure can (`__toString()`).